WooCommerce/src/main/kotlin/com/woocommerce/android/ui/products/variations/VariationsBulkUpdateInventoryViewModel.kt

```diff
@@ -64,8 +64,13 @@ class VariationsBulkUpdateInventoryViewModel @Inject constructor(
         }
     }
 
-    fun onStockQuantityChanged(quantity: Double) {
-        viewState = viewState.copy(stockQuantity = quantity)
+    fun onStockQuantityChanged(rawQuantity: String) {
+        val quantity = rawQuantity.toDoubleOrNull()
+        viewState = if (quantity == null) {
+            viewState.copy(stockQuantity = null, isDoneEnabled = false)
+        } else {
+            viewState.copy(stockQuantity = quantity, isDoneEnabled = true)
+        }
```

Comment:
If you think is relevant, it's possible to simplify this viewState assignment even further:

```suggestion
        viewState = rawQuantity.toDoubleOrNull()?.let { quantity ->
            viewState.copy(stockQuantity = quantity, isDoneEnabled = true)
        } ?: viewState.copy(stockQuantity = null, isDoneEnabled = false)
```